From d7626da004ccd4464169f35a41984be366edfed2 Mon Sep 17 00:00:00 2001
From: rbuj <robert.buj@gmail.com>
Date: Sat, 1 Jan 2022 11:04:26 +0100
Subject: [PATCH 6/6] mate-dictionary: fix memory leak

---
 mate-dictionary/src/gdict-window.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mate-dictionary/src/gdict-window.c b/mate-dictionary/src/gdict-window.c
index 067a2534..96168286 100644
--- a/mate-dictionary/src/gdict-window.c
+++ b/mate-dictionary/src/gdict-window.c
@@ -130,7 +130,6 @@ gdict_window_dispose (GObject *gobject)
   if (window->settings != NULL)
     {
       g_object_unref (window->settings);
-
       window->settings = NULL;
     }
 
@@ -189,6 +188,8 @@ gdict_window_dispose (GObject *gobject)
       window->busy_cursor = NULL;
     }
 
+  g_clear_pointer (&window->sidebar_page, g_free);
+
   G_OBJECT_CLASS (gdict_window_parent_class)->dispose (gobject);
 }
 
-- 
2.36.1

