From 34e381a2f7b7319d0befa3040a5c3b5db900f49d Mon Sep 17 00:00:00 2001
From: rbuj <robert.buj@gmail.com>
Date: Thu, 21 Oct 2021 19:38:43 +0200
Subject: [PATCH 4/6] gsearchtool: fix memory leak

---
 gsearchtool/src/gsearchtool-support.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gsearchtool/src/gsearchtool-support.c b/gsearchtool/src/gsearchtool-support.c
index adffd25f..7e4745fe 100644
--- a/gsearchtool/src/gsearchtool-support.c
+++ b/gsearchtool/src/gsearchtool-support.c
@@ -446,10 +446,10 @@ get_file_type_description (const gchar * file,
 	}
 
 	if (content_type == NULL || g_content_type_is_unknown (content_type) == TRUE) {
-		return g_strdup (g_content_type_get_description ("application/octet-stream"));
+		return g_content_type_get_description ("application/octet-stream");
 	}
 
-	desc = g_strdup (g_content_type_get_description (content_type));
+	desc = g_content_type_get_description (content_type);
 
 	if (g_file_info_get_is_symlink (file_info) == TRUE) {
 
-- 
2.36.1

