From 74646513584b8bcbea61b37ddb3e75a5a206605c Mon Sep 17 00:00:00 2001
From: rbuj <robert.buj@gmail.com>
Date: Mon, 8 Nov 2021 15:26:58 +0100
Subject: [PATCH 1/6] mate-screenshot: fix memory leak

---
 mate-screenshot/src/mate-screenshot.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/mate-screenshot/src/mate-screenshot.c b/mate-screenshot/src/mate-screenshot.c
index e88fc6c4..de959406 100644
--- a/mate-screenshot/src/mate-screenshot.c
+++ b/mate-screenshot/src/mate-screenshot.c
@@ -1180,8 +1180,10 @@ load_options (void)
   /* Find various dirs */
   last_save_dir = g_settings_get_string (settings,
                                          LAST_SAVE_DIRECTORY_KEY);
-  if (!last_save_dir || !last_save_dir[0])
+
+  if (*last_save_dir == '\0')
     {
+      g_free (last_save_dir);
       last_save_dir = get_desktop_dir ();
     }
   else if (last_save_dir[0] == '~')
-- 
2.36.1

